README.md

```diff
@@ -145,14 +198,19 @@ id                 Convert jobid(s) to another form
 overlay            Show flux overlay network status
 kvs                Flux key-value store utility
 pstree             display job hierarchies
+resource           List/manipulate Flux resource status
 shutdown           Shut down a Flux instance
 start              bootstrap a local Flux instance
 logs               Show Flux instance start and stop times
 top                Display Running Flux Jobs
 uptime             Tell how long Flux has been up and running
 ```
 
-**Question for Alec** maybe kvs should just be db or similar?
```

Comment:
I think we should keep KVS the same. It doesn't appear to be fully a database and I think users might expect more out of it if we named it as such. For now it's a simple key value store for flux jobs to use.

Reply:
Agree.